tests/qir/test_pytket_qir_quantum_v.ll

```diff
@@ -0,0 +1,77 @@
+; ModuleID = 'test_pytket_qir_quantum_v'
+source_filename = "test_pytket_qir_quantum_v"
+
+%Qubit = type opaque
+%Result = type opaque
+
+@0 = internal constant [2 x i8] c"c\00"
+
+define void @main() #0 {
+entry:
+  %0 = call i64 @reg2var(i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false, i1 false)
+  call void @__quantum__qis__h__body(%Qubit* null)
+  call void @__quantum__qis__x__body(%Qubit* null)
+  call void @__quantum__qis__y__body(%Qubit* null)
+  call void @__quantum__qis__z__body(%Qubit* null)
+  call void @__quantum__qis__rx__body(double 5.000000e-01, %Qubit* null)
+  call void @__quantum__qis__rzz__body(double 5.000000e-01, %Qubit* null, %Qubit* inttoptr (i64 1 to %Qubit*))
+  call void @__quantum__qis__phasedx__body(double 5.000000e-01, double 4.000000e-01, %Qubit* inttoptr (i64 1 to %Qubit*))
+  call void @__quantum__qis__rxxyyzz__body(double 0x3FF921FB54442D18, double 0x3FF41B2F769CF0E0, double 0x3FEE28C731EB6950, %Qubit* null, %Qubit* inttoptr (i64 1 to %Qubit*))
+  call void @__quantum__qis__rxxyyzz__body(double 0x4012D97C7F3321D2, double 0x401197C987C952C4, double 0x40105616905F83B6, %Qubit* inttoptr (i64 2 to %Qubit*), %Qubit* inttoptr (i64 1 to %Qubit*))
```

Comment:
I am a bit surprised by this float to string conversion in the file, but it seams to be working right.